assets/src/hooks/useShapes.ts

```diff
@@ -56,7 +56,7 @@ export const useTripShape = (tripId: TripId | null): Shape[] => {
         setShape(shapeResult)
       )
     }
-  }, [])
+  }, [tripId])
```

Comment:
With React Hooks Testing Library, it's actually possible to test that this behaves in the way that you want it to. See the `useSwings` hook tests for examples that show how to test that the hook doesn't re-fire when the requested routes don't change, but that it does re-fire when they do change.